tests/test_parser.py

```diff
@@ -323,6 +320,19 @@ def test_login_page_has_expected_title(self):
         actual = validate_title(html)
         self.assertTrue(actual)
 
+    @patch('smarsy.parse.get_user_credentials',
+           return_value={'username': 'login'})
+    def test_childs_page_has_expected_username(self, mocked_value):
+        response_string = '\n<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" \
```

Comment:
there is no need to create such big test strings, 1-2 lines is enough for test

Reply:
OK